### DISCUSSION TITLE
Couldn't use bat as PAGER/MANPAGER (unexpected EOF while looking for matching `'')

### DISCUSSION THREAD

**Discussion by Animeshz** [sharkdp/bat#2370] [Q&A]
```bash
$ MANPAGER="bash -c 'col -b | bat -l man -p'" man bat
-b: -c: line 1: unexpected EOF while looking for matching `''
-b: -c: line 2: syntax error: unexpected end of file
```

Seems like `|` there is acting weird and taking priority over quoted string with single quote `'`.

This however works perfectly fine:

```bash
$ man bat | bash -c 'col -b | bat -l man -p'
```

Any idea why former one with env variable doesn't work?

**Comment by sharkdp** [comment]
I'm not sure. Maybe it depends on your `man` implementation? I can not reproduce this on Arch Linux (man 2.11.0).
